near/test/algoHelpers.ts

```diff
@@ -0,0 +1,435 @@
+import algosdk, {
```

Comment:
Just out of curiosity, why is there Algorand stuff in the Near folder?  Are they related in some way?

Reply:
allowed the tests to work